.github/workflows/ui_tests.yml

```diff
@@ -33,17 +33,29 @@ jobs:
           node-version: 18.x
           cache: 'npm'
           cache-dependency-path: 'package-lock.json'
+      - name: Get installed Playwright version
+        id: playwright-version
+        run: echo "PLAYWRIGHT_VERSION=$(node -e "console.log(require('./package-lock.json').packages['apps/laboratory']['devDependencies']['@playwright/test'])")" >> $GITHUB_ENV
```

Comment:
Maybe we could abstract this to a script to mantain it more easily. Also, we could maybe simplify it by using `npm list` + grep. Something like `npm list @playwright/test | grep /^@/` or similar. Not sure if it works as flawlessly as this though